google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -119,6 +119,7 @@ appengine.deployment.source.not.found.error=Deployment source not found: {0}.
 appengine.deployment.invalid.source.error=Invalid deployment source.
 appengine.deployment.status.deploying=Deploying to App Engine
 appengine.deployment.error.invalid.cloudsdk=Invalid Cloud SDK directory path.
+appengine.deployment.deployallconfigs.checkbox.label=Deploy all App Engine service configuration files (indexes, dos, dispatch, cron, queues)
```

Comment:
How about something like:

"Include all App Engine service configuration files (..) with the deployment"

Just to make it totally clear that were're still deploying the artifact.